lib/block.mli

```diff
@@ -66,7 +118,19 @@ val pp_line_directive: (string * int) Fmt.t
 
 (** {2 Accessors} *)
 
-val mode: t -> [`Non_det of [`Command|`Output] | `Normal]
+val line: t -> int
+
+val filename: t -> string
+
+val section: t -> section option
+(** [section t] is [t]'s section. *)
+
+val contents: t -> string list
+
+val value: t -> value
+(** [value t] is [t]'s value. *)
+
+val mode: t -> [`Command | `Output] option
 (** [mode t] is [t]'s mode. *)
```

Comment:
`` `Non_det of .. `` was the mode, ``[`Command | `Output]`` was only the argument of that mode.
It's true that we never needed the "mode" concept. This function should be renamed `non_det` or similarly.

Reply:
Done